clouddriver-web/src/main/groovy/com/netflix/spinnaker/clouddriver/controllers/ServerGroupController.groovy

```diff
@@ -142,6 +142,9 @@ class ServerGroupController {
         sg.instanceType = serverGroup.launchConfig.instanceType
       }
     }
+    if (serverGroup.launchTemplate && serverGroup.launchTemplate.launchTemplateData) {
+      instanceType = serverGroup.launchTemplate.launchTemplateData.instanceType
```

Comment:
`instanceType` isn't actually defined anywhere, I think you actually want `sg.instanceType` (but if there was a `getInstanceType()` on `AmazonServerGroup`, you wouldn't need this).